Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
```

Comment:
Does it makes sense to add README for such a project?

Reply:
yes indeed, I will add with next PR

Reply:
added in this one :see_no_evil: